### PR TITLE
Improvements in the welcome message

### PR DESCRIPTION
**Pull request by benjagm**
Added Improvements in the welcome message to encourage new members to introduce themselves.

**GitHub Issue:** # / NA

**Summary**:

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

Yes/No
[JUSTIFICATION]

### CODE REVIEW

File: slack/greeting [hunk 1]
```diff
@@ -1,19 +1,19 @@
-Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+:wave: Hey {{eec4a322-082c-4247-8cdd-b67d5eeee6aa==user}}, Welcome to the JSON Schema Slack Server!
 
 Whatever the reason you chose to join us, we want you to know, we're glad you're here.
 
-We have a formal [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md) which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
+We have a formal Code of Conduct which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
```
Comment (Julian) [inline slack/greeting:5]: Doesn't this remove the link to the Code of Conduct?

File: slack/greeting [hunk 2]
```diff
@@ -1,19 +1,19 @@
-Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+:wave: Hey {{eec4a322-082c-4247-8cdd-b67d5eeee6aa==user}}, Welcome to the JSON Schema Slack Server!
```
Comment (Julian) [inline slack/greeting:1]: I don't know what's responsible for interpolating these variables, but is the change here from `{{Joiner}}` to this random looking UUID correct?

File: slack/greeting [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
-We have a formal [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md) which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
+We have a formal Code of Conduct which we invite you to review. It is short, and reading it is respectful to the community you've chosen to enter.
 
 If you need help, or you're looking to get involved with the JSON Schema community, you're in the right place.
 
 This server includes the specification authors, many implementers, and a large number of helpful individuals from the community.
 
-If you're looking for an implementation of JSON Schema, you should check out the [implementations page](https://json-schema.org/implementations.html).
+:mag: If you're looking for an implementation of JSON Schema, you should check out the implementations page.
```
Comment (Julian) [inline slack/greeting:11]: This looks like it removes a bunch of links here too. I'm assuming all that's an accident, no?

**Review by Julian — CHANGES_REQUESTED** [review]

File: slack/greeting (hunk 2 above)
Comment (Relequestual) [reply slack/greeting:1]: I would say no. I THINK when I copied it here originally, I had to manually replace those.
It is Slack that's responsible for those ID thinggy in that case.

**Comment by gregsdennis** [comment]
I agree with Julian that the links should be there.

File: slack/greeting (hunk 1 above)
Comment (benjagm) [reply slack/greeting]: This was a copy/paste issue when copying from Workflow builder. Fixed.

File: slack/greeting (hunk 2 above)
Comment (benjagm) [reply slack/greeting]: Fixed.

File: slack/greeting (hunk 3 above)
Comment (benjagm) [reply slack/greeting]: This was a copy/paste issue when copying from Workflow builder.
Fixed.

**Comment by benjagm** [comment]
Thanks for the feedback. All the links were missing because of a copy/paste issue.

Everything has been fixed with the last push.

**Review by Julian — APPROVED** [review]
lgtm now!

**Review by Relequestual — APPROVED** [review]

**Pull request merged**